### DISCUSSION TITLE
ExcelUtil导入excel单元格为时间日期，获取时变成当前时间

### DISCUSSION THREAD

**Discussion by 973048054** [looly/hutool#2937] [Q&A]
导入的数据：
![image](https://user-images.githubusercontent.com/83212229/222070342-be272036-01c9-4b1e-8a0d-a98d71be7826.png)
单元格格式：
![image](https://user-images.githubusercontent.com/83212229/222070970-e871ea2d-80fd-48ba-92f9-52ee0c1b50fa.png)

获取到的数据：
![image](https://user-images.githubusercontent.com/83212229/222070750-d0dc6b1f-603d-4044-b5b5-93eb05ed509d.png)

**Comment by looly** [comment]
麻烦提个issue，提供下测试数据。我看是否是日期格式的问题。

**Reply by 973048054** [reply]
提交issue了

**Reply by 973048054** [reply]
测试数据:

[Uploading 导出文件1677649503673.xlsx…]()
